.changeset/rich-colts-laugh.md

```diff
@@ -0,0 +1,41 @@
+---
+"@comet/cms-site": major
```

Comment:
This should be a minor change

Reply:
done in f2796936a7df27e4958c127ebbff6755cfbfbedd